ui/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "npm test --workspaces",
     "lint": "npm run lint --workspaces",
     "lint:fix": "npm run lint:fix --workspaces",
-    "start": "npm run start -w app",
+    "start": "npm run clean && npm run start -w app",
```

Comment:
will there be cases when we don't want to run clean before start? is it better to have a separate command that runs both?